### PR TITLE
Kogito 7211 aj tweaks

### PR DESCRIPTION
**Pull request by AjayJagan**
**JIRA:**

**Description:**

Please make sure that your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc [hunk 1]
```diff
@@ -32,8 +32,11 @@ For more information about the tooling and the required dependencies, see xref:g
 To create your Serverless Workflow service, first you need to bootstrap a project.
```
Comment (ricardozanini) [inline serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc:32]: In the `Prerequisites`, please mention that either Quarkus CLI or KN CLI can be used.

**Review by ricardozanini — COMMENTED** [review]
Just one minor thing :)

File: serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ image::getting-started/hello-world-workflow.png[]
 .Prerequisites
 * Java {java_min_version} is installed with `JAVA_HOME` configured appropriately.
 * Maven {maven_min_version}
-* {quarkus_cli_url}[Quarkus CLI] {quarkus_version}
+* {quarkus_cli_url}[Quarkus CLI] {quarkus_version} or xref:tooling/kn-plugin-workflow/kn-plugin-workflow-overview.adoc[Knative CLI] {kn-cli-version}
```
Comment (AjayJagan) [inline serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc:23]: Hi @ricardozanini  Added the note for KN-CLI here.. let me know if it is good :)

**Review by ricardozanini — APPROVED** [review]
Great! Now let's try to merge everything together. 😅

**Review by paulovmr — APPROVED** [review]

**Pull request merged**